hack/python-sdk/gen-sdk.sh

```diff
@@ -28,8 +28,22 @@ if [ -z "${GOPATH:-}" ]; then
     export GOPATH=$(go env GOPATH)
 fi
 
+# Grab kube-openapi version from go.sum
+OPENAPI_VERSION=$(grep 'k8s.io/kube-openapi' go.sum | awk '{print $2}' | head -1)
+OPENAPI_PKG=$(echo `go env GOPATH`"/pkg/mod/k8s.io/kube-openapi@${OPENAPI_VERSION}")
+
+if [[ ! -d ${OPENAPI_PKG} ]]; then
+    echo "${OPENAPI_PKG} is missing. Running 'go mod download'."
+    go mod download
+fi
+
+echo ">> Using ${OPENAPI_PKG}"
+
+echo "Building openapi-gen"
+go build -o openapi-gen ${OPENAPI_PKG}/cmd/openapi-gen
+
 echo "Generating OpenAPI specification ..."
-go run vendor/k8s.io/code-generator/cmd/openapi-gen/main.go --input-dirs github.com/kubeflow/pytorch-operator/pkg/apis/pytorch/v1,github.com/kubeflow/common/job_controller/api/v1 --output-package github.com/kubeflow/pytorch-operator/pkg/apis/pytorch/v1 --go-header-file hack/boilerplate/boilerplate.go.txt
```

Comment:
@jinchihe  Do we need to add`github.com/kubeflow/common/job_controller/api/v1` as input? I actually have some failure if I add common API `github.com/kubeflow/common/job_controller/api/v1` like below. Have you ever seen the problem? 

```
$ ./openapi-gen --input-dirs github.com/kubeflow/common/job_controller/api/v1,github.com/kubeflow/pytorch-operator/pkg/apis/pytorch/v1 --output-package github.com/kubeflow/pytorch-operator/pkg/apis/pytorch/v1 --go-header-file hack/boilerplate/boilerplate.go.txt

E0519 14:11:58.053066   14182 openapi.go:294] Error when generating: PodInformerSynced, PodInformerSynced k8s.io/client-go/tools/cache.InformerSynced
2020/05/19 14:11:58 OpenAPI code generation error: Failed executing generator: some packages had errors:
cannot generate spec for type k8s.io/client-go/tools/cache.InformerSynced

```

Reply:
cc @jinchihe

Reply:
Sorry I missed the PR since I was on vacation last month, will take a look asap. Thanks,

Reply:
> @jinchihe Do we need to add`github.com/kubeflow/common/job_controller/api/v1` as input? I actually have some failure if I add common API `github.com/kubeflow/common/job_controller/api/v1` like below. Have you ever seen the problem?
> 
> ```
> $ ./openapi-gen --input-dirs github.com/kubeflow/common/job_controller/api/v1,github.com/kubeflow/pytorch-operator/pkg/apis/pytorch/v1 --output-package github.com/kubeflow/pytorch-operator/pkg/apis/pytorch/v1 --go-header-file hack/boilerplate/boilerplate.go.txt
> 
> E0519 14:11:58.053066   14182 openapi.go:294] Error when generating: PodInformerSynced, PodInformerSynced k8s.io/client-go/tools/cache.InformerSynced
> 2020/05/19 14:11:58 OpenAPI code generation error: Failed executing generator: some packages had errors:
> cannot generate spec for type k8s.io/client-go/tools/cache.InformerSynced
> ```

@Jeffwan I do not remember hit the problem. If the problem can be resolved by adding the input , yeah, please go ahead.